src/antsibull_docs/process_docs.py

```diff
@@ -63,36 +138,45 @@ def normalize_plugin_info(
 
     errors: list[str] = []
     if plugin_type == "role":
+        role_schema: type[BaseModel] = DOCS_SCHEMAS[
+            plugin_type
+        ]  # type: ignore[attr-defined, assignment]
```

Comment:
Why does this need a `type: ignore`? Is `DOC_SCHEMAS` not typed correctly?

Reply:
`plugin_type` is not typed correctly. It's an arbitrary string at this point.